plugins/gateway/kong/plugin.go

```diff
@@ -32,6 +32,7 @@ import (
 )
 
 type Config struct {
+	EnableTLS bool
```

Comment:
please add `json` flag, must be the same as in the [plugin config file](https://github.com/apiclarity/apiclarity/blob/master/plugins/gateway/kong/deploy/kongPlugin.yaml)